Backend/app/provider-platform/dynamic-offer-driver-app/Main/src/Storage/Queries/OperationHubRequestsExtra.hs

```diff
@@ -45,26 +49,29 @@ findAllRequestsInRange from to limit offset mbMobileNumberHash mbReqStatus mbReq
           B.limit_ (toInteger limit) $
             B.offset_ (toInteger offset) $
               B.filter_'
-                ( \(operationHubRequests, driver) ->
+                ( \(operationHubRequests, driver, operationHub) ->
                     B.sqlBool_ (operationHubRequests.createdAt B.>=. B.val_ from)
                       B.&&?. B.sqlBool_ (operationHubRequests.createdAt B.<=. B.val_ to)
-                      B.&&?. maybe (B.sqlBool_ $ B.val_ True) (\driverId -> operationHubRequests.driverId B.==?. B.val_ driverId) mbDriverId
+                      B.&&?. maybe (B.sqlBool_ $ B.val_ True) (\driverId -> operationHubRequests.creatorId B.==?. B.val_ driverId) mbDriverId
```

Comment:
could you rename `driverId` to `creatorId`?

`B.&&?. maybe (B.sqlBool_ $ B.val_ True) (\creatorId -> operationHubRequests.creatorId B.==?. B.val_ creatorId) mbCreatorId`